django/Dockerfile

```diff
@@ -76,6 +80,8 @@ RUN chown django:django ./supervisord.conf \
 
 # Define environment variables
 ENV PYTHONPATH="/omniport:/omniport/core:/omniport/services:/omniport/apps"
+ENV HISTFILE="/.history/.bashrc"
```

Comment:
I think history is stored in `.bash_history` not `.bashrc`. Please confirm.

Reply:
Oh, I missed that. Yeah it works it's just any file, but I will push it after the change.